auto_update.py

```diff
@@ -0,0 +1,53 @@
+name: Update NVIDIA GRID Driver
```

Comment:
this files seems to contain yaml, this isn't a valid Python file.

Reply:
oops, I copied the yaml from my fork twice, instead of the python code: https://github.com/ganeshkumarashok/aks-gpu